src/EdFi.AnalyticsMiddleTier.DataStandard32/Base/PostgreSQL/0036-View-StudentLocalEducationAgencyDim-Alter.sql

```diff
@@ -0,0 +1,81 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE OR REPLACE VIEW analytics.StudentLocalEducationAgencyDim
+AS
+   SELECT 
+           CONCAT(Student.StudentUniqueId, '-', LocalEducationAgency.LocalEducationAgencyId) AS StudentLocalEducationAgencyKey,
+           Student.StudentUniqueId AS StudentKey,
+           CAST(LocalEducationAgency.LocalEducationAgencyId AS VARCHAR) AS LocalEducationAgencyKey, 
+           Student.FirstName AS StudentFirstName, 
+           COALESCE(Student.MiddleName, '') AS StudentMiddleName, 
+           Student.LastSurname AS StudentLastName, 
+           COALESCE(LimitedEnglishProficiencyDescriptor.CodeValue, 'Not Applicable') AS LimitedEnglishProficiency, 
+           COALESCE(StudentEducationOrganizationAssociation.HispanicLatinoEthnicity, false) AS IsHispanic, 
```

Comment:
Here and several and in a bunch of join conditions:

```suggestion
           COALESCE(studentEdOrg.HispanicLatinoEthnicity, false) AS IsHispanic, 
```

Or vice versa, just remove the alias on line 34, which doesn't do much to aid comprehension anyway.